ui/src/router/index.js

```diff
@@ -40,7 +40,17 @@ router.beforeEach((to, from, next) => {
     } else {
       next();
     }
-  } else {
+  }
+  else if (to.path=="/login")
+  {
+    if (isAuthenticated) {
```

Comment:
We can check if the path is `/login` and the user is authenticated in the same line.